src/main/include/subsystems/Superstructure.h

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) FRC Team 2363. All Rights Reserved.
+
+#pragma once
+
+#include <frc/DoubleSolenoid.h>
+#include <frc/controller/PIDController.h>
+#include <frc/DutyCycleEncoder.h>
+#include <frc2/command/SubsystemBase.h>
+#include <rev/CANSparkMax.h>
+#include <rev/SparkMaxLimitSwitch.h>
+#include <units/angle.h>
+
+#include "Constants.h"
+#include "frc/DutyCycleEncoder.h"
+
+using namespace ElectricalConstants;
+
+class Superstructure : public frc2::SubsystemBase {
+ public:
+  Superstructure();
+
+  void SetIntakeWheelSpeed(double speed);
+
+  void SetArmPosition(units::radian_t position);
+
+  units::radian_t GetArmPosition();
+
+  bool HasGamePiece();
+
+  void SuperstructurePeriodic();
```

Comment:
`SuperstructurePeriodic()` could be reduced to `Periodic()` to avoid redundancy.

Reply:
`SuperstructurePeriodic() ` is run in TimesliceRobot for the 5ms loop time instead of 20ms normal frequency of `Periodic()`